### PR TITLE
Desktop-version

### PR DESCRIPTION
**Pull request by BonkeGcobo**
We created the mobile version of the project

### CODE REVIEW

File: css/myStyle.css [hunk 1]
```diff
@@ -271,3 +279,229 @@ form {
...
+
+.desktop_title {
+  display: none;
+}
+
+.desk_var {
+  display: none;
+}
+
+@media only screen and (min-width: 915px) {
```
Comment (SajjadAhmad14) [inline css/myStyle.css]: Please make sure the value for min-width property should be in range of 992px - 1100px.

File: css/myStyle.css [hunk 2]
```diff
@@ -31,10 +31,11 @@ body {
 
 .canop {
   list-style: none;
+  color: #344563;
 }
 
 .headline {
```
Comment (SajjadAhmad14) [inline css/myStyle.css:37]: Please use the same font color for headline text and `let's connect` text. Moreover, there is a round border on the bottom left of the section in the original design. Kindly see the below screenshot and fix issues accordingly.
intro

File: Portfolio.html [hunk 3]
```diff
@@ -64,189 +69,269 @@ <h1 class="intro_2">
   </div>
 
   <section class="Works">
```
Comment (SajjadAhmad14) [inline Portfolio.html:71]: Please add a border to project cards as it's in the design. Moreover please use `Poppins font for all the whole content and use the same color from the design.

File: Portfolio.html [hunk 4]
```diff
@@ -64,189 +69,269 @@ <h1 class="intro_2">
...
+    <div class="About_me">
+      <div class="myself_title">
       <h1 class="abt_myself_title about_one">
         About
       </h1>
       <h1 class="abt_myself_title about_two">
         Myself
       </h1>
-    </div>
+      </div>
```
Comment (SajjadAhmad14) [inline Portfolio.html]: The `About me` section should have a button to get the resume. Moreover, please use the same font color from the design.

File: Portfolio.html [hunk 5]
```diff
@@ -283,20 +370,23 @@ <h1 class="sec_header my_skills">Skills</h1>
...
       </li>
-    </ul>
+      </ul>
+    </div>
   </section>
 
   <footer class="contact_form">
     <h1 class="contact_form_heading">
       Contact Form
     </h1>
```
Comment (SajjadAhmad14) [inline Portfolio.html:386]: Contact form heading should be in the center of the form. Please see the below screenshot for reference.
contact

File: Portfolio.html [hunk 6]
```diff
@@ -25,7 +25,12 @@
       </div>
```
Comment (SajjadAhmad14) [inline Portfolio.html:25]: Please add a project description of the features and add instructions to set the project up locally in README. Moreover, PR should have a descriptive summary of project features.

File: css/myStyle.css [hunk 7]
```diff
@@ -271,3 +279,229 @@ form {
 .btn_submit:hover {
   cursor: pointer;
 }
```
Comment (SajjadAhmad14) [inline css/myStyle.css:284]: [ OPTIONAL ] Please make an assets folder and put all of your images and other assets in it. The project directory will look nice and managed.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Bonke Gcobo, 
 ### You have been doing a tremendous job so far :ok_hand:
## To Highlight 🎉
- [x] GitHub flow is used properly.
- [x] Linter checks are passing.
## To Improve 🙏
- [x] Please check inline comments under the review
## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: css/myStyle.css (hunk 2 above)
Comment (BonkeGcobo) [reply css/myStyle.css:37]: Okay thank you, Sajjad however in the figma design the let's connect and the headline text have different colors

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Bonke Gcobo, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏

## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**